.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+Notes:
+* Pull requests will not be accepted until the submitter has agreed to the contributer agreement.
```

Comment:
should link point to:

https://github.com/ARMmbed/mbed-os/blob/master/CONTRIBUTING.md
or
http://developer.mbed.org/contributor_agreement/